eds4jinja2/main.py

```diff
@@ -1,11 +1,13 @@
+import pathlib
 from distutils.dir_util import copy_tree
 
 from eds4jinja2.builders.report_builder import ReportBuilder
 import click
 
 
 def copy_static_content(from_path, to_path):
-    copy_tree(from_path,to_path)
+    if pathlib.Path(from_path).is_dir():
```

Comment:
what happens if it's not a `dir`? _(where it's used)_

Reply:
well, hard fail, is also a resolution.